gk/main.c

```diff
@@ -1643,8 +1643,12 @@ process_pkts_back(uint16_t port_back, uint16_t port_front,
 
 		fib = look_up_fib(&gk_conf->lpm_tbl, &packet.flow);
 
-		 /* No entry for the destination, drop the packet. */
-		if (fib == NULL) {
+		 /*
+		  * No entry for the destination or the entry instructs
+		  * to forward the packet to the neighbor in the back network,
+		  * drop the packet.
+		  */
```

Comment:
1. We can drop this comment. The first two lines are just reading the test and the last line is not accurate since we might redirect the packet instead of dropping.

2. I can't comment on the line with `"gk: failed to get the fib entry"` below, but we need to improve this message to `"gk: failed to get the fib entry or it is not an IP packet"`.